sensu/plugins/metrics-nova-project-quotas.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import sys
+import time
+import traceback
+
+import shade
+import novaclient
+
+OK = 0
+WARNING = 1
+CRITICAL = 2
+UNKNOWN = 3
+
+
+class CloudMetrics(object):
+    def __init__(self):
+        self.cloud = shade.openstack_cloud()
+
+    def get_projects(self):
+        return {
+            project['id']: project['name']
+            for project in self.cloud.search_projects()
```

Comment:
It's essentially the same thing.  `search_projects` calls `list_projects`, this just provides filtering.